cypress/e2e/signIn.cy.js

```diff
@@ -11,28 +11,32 @@ describe('Sign In page', () => {
   let user;
 
   before(() => {
-    cy.task('db:clear');
     cy.task('generateUser').then(generateUser => {
       user = generateUser;
     });
   });
 
-  it('should provide an ability to log in with existing credentials', () => {
-    signInPage.visit();
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.wait(500);
```

Comment:
Avoid arbitrary waits in Cypress tests. When cypress tasks are used, waits should not be necessary as cypress handles them automatically.